charts/tfy-clickhouse/templates/clickhouse-installation.yaml

```diff
@@ -91,6 +91,17 @@ spec:
           imagePullSecrets:
               {{- toYaml . | nindent 12 }}
           {{- end }}
+          volumes:
+            - name: tmp
+              emptyDir: {}
+          securityContext:
+            capabilities:
+              add:
+                - CAP_NICE
+                - CAP_IPC_LOCK
+            fsGroup: 101
+            runAsGroup: 101
+            runAsUser: 101
```

Comment:
This can potentially create an issue with the existing data on EBS.
Please test on Devtest.

Reply:
Check if Clickhouse can load the existing data once you apply this change.

Reply:
Let's stick with `1000` as that is pretty common unless we have good reasons for 101